### DISCUSSION TITLE
Filtering object within the array

### DISCUSSION THREAD

**Discussion by sfakhrym** [quasarframework/quasar#9155] [General - Components / Directives / etc] [answered]
Hello Quasar friends :)

I want to ask about filtering the object inside the array.

If I have 3 variables (focuses1, focuses2, focuses3)

```
const focuses1 = reactive([{
      title: 'Your Focus',
      desc: 'Description',
      edit: false,
      priority: ['1st','2nd','3rd']
    }])
```

```
const focuses2 = reactive([{
      title: 'Your Focus',
      desc: 'Description',
      edit: false,
      priority: ['1st','2nd','3rd']
    }])
```

```
const focuses3 = reactive([{
      title: 'Your Focus',
      desc: 'Description',
      edit: false,
      priority: ['1st','2nd','3rd']
    }])
```

I want to filter only the array with the property of `priority` with value of '1st', or `priority[0]`

I have tried with computed method using filter function, but I got confuse with how the syntax works :'D...

Or is there any other method, guys?

Thanks in advance for helping :)

**Comment by metalsadman** [comment, marked as answer]
```js
const something = computed(() => focuses3.filter(v => v.priority[0] === '1st'))
//or
const something2 = computed(() => focuses3.filter(v => v.priority.some(j => j === '1st')))
```
https://v3.vuejs.org/api/computed-watch-api.html#computed
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

**Reply by sfakhrym** [reply]
Thanks, Sir for helping :)..
